docs/documentation/maya/muscle.md

```diff
@@ -111,108 +70,117 @@ Following, we present the key aspects to keep in mind while painting each attrib
 
-Enabling the debugger and selecting one of these constraints will draw lines from the influenced vertices in the simulated mesh to their corresponding reference vertices. 
+Enabling the debugger and selecting one of these constraints will draw lines from the influenced vertices in the simulated mesh to their corresponding reference vertices.
 
-### Debugger attributes
+<figure markdown>
+  ![AdnMuscle debug](images/muscle_debug.png)
+  <figcaption><b>Figure 6:</b> AdnMuscle being debugged (from left ro right, "Muscle Fibers", "Attachment Constraints" and "Slide On Segment")</figcaption>
+</figure>
 
-The following attributes can be modified to better customize the appereance of these lines:
+# Advanced
 
- - **Width Scale** (Float, 1.0): Modifies the width of all lines.
- - **Color** (Color picker): Selects the line color from a color wheel. Its saturation can be modified using the slider.
+## Attachments
 
-Additionally, for the specific case of debugging fibers, one more attribute is available:
+Once the AdnMuscle deformer is created, it is possible to add and remove new attachments to the system. 
 
- - **Fiber scale value** (Float, 3.0): can be modified to set a custom fiber length.
+- **Add attachments**:  
+    1. Select the transform nodes (one or more) that you want to assign as attachments to the Muscle.
+    2. Select the mesh that has the AdnMuscle deformer applied.
+    3. Press the ![Add Attachments button](images/adn_add_attachment.png){width=40px} button in the AdonisFX shelf or press "Add Attachments" in the AdonisFX menu from the Edit Muscle submenu.
+- **Remove attachments**:
+    1. Select one or more transform nodes that are assigned as attachments to the Muscle.
+    2. Select the mesh that has the AdnMuscle deformer applied.
+    3. Press the ![Remove Attachments button](images/adn_remove_attachment.png){width=40px} button in the AdonisFX shelf or press "Remove Attachments" in the AdonisFX menu from the Edit Muscle submenu. 
+    4. Alternatively, if only the mesh with the AdnMuscle deformer is selected, when pressing the ![Remove Attachments button](images/adn_remove_attachment.png){width=40px} button, all attachments will be removed.
 
-<figure markdown>
-  ![Muscle editor debug menu](images/attribute_editor_muscle_debug.png)
-  <figcaption>Figure 4: Muscle Attribute Editor (Debug menu)</figcaption>
-</figure>
+## Slide on Segment Constraint
+
+Additionally to all previously mentioned constraints, muscles can have an additional, optional constraint. Here the user may add attachments that will define a segment over which the muscle will slide.
+
+- **Add Segment**:
+    1. Select the transform nodes (two or more if this is the first segment you are adding) from which you would wish to make a segment for the muscle to slide on.
+    2. Select the mesh that has the AdnMuscle deformer applied.
+    3. Press "Add Slide On Segment Constraint" in the AdonisFX menu from the Edit Muscle submenu.
+- **Remove Segment**:  
```

Comment:
Please add the same comment as for ribbon if it's not already there (I can't see it on Git):

```
> [!NOTE]
> - The transform nodes selection must follow a parent to child relationship in the hierarchy.
```

Reply:
Done